### DISCUSSION TITLE
How to start test in custom directory?

### DISCUSSION THREAD

**Discussion by vit1251** [mesonbuild/meson#9680] [Q&A] [answered]
How to make test execution in a custom directory with resources, i.e. config file for example.

I expect some like that
```
test('Check valida config', executable('test_config', 'test/test_config.c', working_directory='./resources'))
```

**Comment by dcbaker** [comment, marked as answer]
There is a `workdir` argument to `test` command, see here: https://mesonbuild.com/Reference-manual.html#test

**Comment by XNerv** [comment]
`workdir` not working on windows? 
Is not working.

For testing purposes only:
    `test('gtest test', e, workdir: 'D:/temp')`

**Comment by eli-schwartz** [comment]
It does work, the CI tests it using `test cases/common/92 test workdir/`
